desktop/apps/artwork/components/auction/helpers.coffee

```diff
@@ -25,6 +26,8 @@ module.exports = {
         'qualified-to-bid'
       else
         'registration-pending'
+    else if moment().isAfter(auction.registration_ends_at)
+      'registration-closed'
```

Comment:
@devangt - Can you view this file and verify that there are not any further paths? Also @erikdstock? I'm not 100% certain of the scope of this behavior.